explorer/client/src/utils/api/rpcSetting.ts

```diff
@@ -39,17 +44,21 @@ export const tryGetRpcSetting = (): string | null => {
 };
 
 const ENDPOINTS = {
-    local: 'http://127.0.0.1:5001',
-    devnet: 'https://gateway.devnet.sui.io:443',
+    [Network.Local]: 'http://127.0.0.1:5001',
+    [Network.Devnet]: 'https://gateway.devnet.sui.io:443',
 };
 
-export function getEndpoint(): string {
+export function getEndpoint(network: Network | string): string {
+    //Endpoint has 3 types:
```

Comment:
```suggestion
    // Endpoint has 3 types:
```
nit: single space after `//`